tsconfig.json

```diff
@@ -10,6 +10,8 @@
     // `transpileOnly` mode for the `ts-node`. This change requires to treat types for each module 
     // independently, which is done by setting the `isolatedModules` flag to `true`.
     "isolatedModules": true,
+    // Detect unused imports, variables, and functions
+    "noUnusedLocals": true,
```

Comment:
It's fine to use other than the cases when some scripts might fail running with `ts-node` during development which have some unused code.

Reply:
are you sure this is an issue with `ts-node`, we have `"transpileOnly": true` set so it shouldn't bother about this

Reply:
Just checked on a test script and with this configuration `ts-node` does not impact. So we are good with it.